WeakAuras/Prototypes.lua

```diff
@@ -2153,7 +2153,6 @@ Private.event_prototypes = {
         end
         local standing
         if tonumber(standingId) then
-           standing = getglobal("FACTION_STANDING_LABEL"..standingId)
```

Comment:
you accidentally removed this line here and readded it in the next commit.

You can fix that with a bit of git-fu.

Start with both git commits applied.
Run "git reset -p HEAD~1"
This will show you each chunk, answer "y" for this chunk, "n" for every other
You can examine what this does via. git diff --staged vs git diff
Next is
"git commit --amend"
"git commit -m squash"

Now git rebase -i HEAD~3
move the squash commit to the second line and replace "pick" with fixup.